### ISSUE TITLE
Button element missing accessible name.

### ISSUE THREAD

**Issue by Sukha11y** [Sukha11y/Test-ZT-fix#3]
Problem
Where is it located
index.html - [https:///c%3A/Users/SukhadaPawar/Test-ZT-fix/Test-ZT-fix/index.html](https://c/Users/SukhadaPawar/Test-ZT-fix/Test-ZT-fix/index.html)

Locate element
XPath or CSS selector
body>div:nth-of-type(2)>*:nth-child(8)

Copy selector to clipboard
HTML element
<button class="icon-only"></button>

Copy HTML element to clipboard
Description
One or more button elements, without an ARIA-assigned role, not intentionally hidden in the DOM and available to assistive technologies, does not have a mechanism that allows an accessible name value to be calculated.
Why it matters
When button elements do not have an accessible name, assistive technologies may incorrectly render the name or provide no name at all to users. When unnamed controls are present, users of assistive technologies may not be able to complete a form.

ettings
Details
Button element missing accessible name.
Critical
WCAG 2.2 Level A - 1.3.1 Info and Relationships

WCAG 2.2 Level A - 2.5.3 Label in Name

WCAG 2.2 Level A - 4.1.2 Name, Role, Value

Problem
Solution
Solution
What to do
Provide an accessible name for all buttons. The text or alternative text of images within buttons is used as the accessible name of the button. When on-screen text is not available, authors can use another method to provide an accessible name, such as by using aria-labelledby, aria-label, or title attribute, to provide the accessible name. The aria-labelledby attribute must contain one or more IDs of elements that label the button. The aria-label attribute is best when there is no on-screen label. Authors should place the string that labels the form field in the aria-label attribute. Authors can also identify an accessible name using an associated label element whenever an on-screen label is present. The label element's value is then explicitly associated with the relevant form field using the for and id attributes. Use of the label element provides an added benefit as it increases the target area that users can click to focus the form field. In certain instances, however, it is impractical or 'undesirable' to use an explicit label for a button; this is largely driven by the fact that the value between the opening and closing label tags must, by definition, be a block of displayed text.
